po/pt.po

```diff
@@ -0,0 +1,914 @@
+
+#. Description used in about dialog.
+#: src/text.vala:293 data/com.github.ztefn.haguichi.appdata.xml.in.in:7
+msgid "A graphical frontend for Hamachi"
+msgstr "Uma interface gráfica para o Hamachi"
+
+#. Generic name used in application launchers.
+#: src/text.vala:295
+msgid "Hamachi Network Manager"
+msgstr "Administrador de Rede Hamachi"
+
+#. Description used in application launchers.
+#: src/text.vala:297
+msgid "Join and create local networks over the Internet"
+msgstr "Junte-se e crie uma rede local via Internet"
+
+#: src/text.vala:299
+msgid "_Close"
+msgstr "_Fechar"
+
+#: src/text.vala:300
+msgid "_Cancel"
+msgstr "_Cancelar"
+
+#: src/text.vala:302
+msgid "_Save"
+msgstr "_Gravar"
+
+#: src/text.vala:303
+msgid "_Revert"
+msgstr "_Reverter"
+
+#: src/text.vala:304
+msgid "_Delete"
+msgstr "_Apagar"
+
+#: src/text.vala:305
+msgid "_Information"
+msgstr "_Informação"
+
+#: src/text.vala:306
+msgid "_Preferences"
+msgstr "_Preferências"
+
+#: src/text.vala:307
+msgid "_Keyboard Shortcuts"
+msgstr "_Atalhos do Teclado"
+
+#: src/text.vala:308
+msgid "_Help"
+msgstr "_Ajuda"
+
+#: src/text.vala:309
+msgid "_About"
+msgstr "_Sobre"
+
+#: src/text.vala:310
+msgid "_Quit"
+msgstr "_Sair"
+
+#: src/text.vala:312
+msgid "_Show Haguichi"
+msgstr "_Mostrar Haguichi"
+
+#: src/text.vala:314
+msgid "Yes"
+msgstr "Sim"
+
+#: src/text.vala:315
+msgid "No"
+msgstr "Não"
+
+#: src/text.vala:317
+msgid "_Hamachi output"
+msgstr "_Resultado do Hamachi"
+
+#: src/text.vala:319
+msgid "IPv4 address:"
+msgstr "Endereço IPv4:"
+
+#: src/text.vala:320
+msgid "IPv6 address:"
+msgstr "Endereço IPv6:"
+
+#: src/text.vala:321
+msgid "Client ID:"
+msgstr "ID de Cliente:"
+
+#: src/text.vala:322
+msgid "Status:"
+msgstr "Estado:"
+
+#: src/text.vala:323
+msgid "Tunnel:"
+msgstr "Túnel:"
+
+#: src/text.vala:324
+msgid "Connection:"
+msgstr "Conexão:"
+
+#: src/text.vala:325
+msgid "Direct"
+msgstr "Direto"
+
+#: src/text.vala:326
+msgid "Relayed"
+msgstr "Retransmitido"
+
+#: src/text.vala:327
+msgid "Anonymous"
+msgstr "Anónimo"
+
+#: src/text.vala:328
+msgid "_Network ID:"
+msgstr "_ID de Rede:"
+
+#: src/text.vala:330
+msgid "_Password:"
+msgstr "_Senha:"
+
+#: src/text.vala:331
+msgid "_Nickname:"
+msgstr "_Alcunha:"
+
+#: src/text.vala:334
+msgid "Offline"
+msgstr "Desligado"
+
+#: src/text.vala:335
+msgid "Online"
+msgstr "Ligado"
+
+#: src/text.vala:336
+msgid "Unreachable"
+msgstr "Inacessível"
+
+#: src/text.vala:337
+msgid "Protocol mismatch"
+msgstr "Protocolo incompatível"
+
+#: src/text.vala:338
+msgid "Conflicting address"
+msgstr "Conflito de endereço"
+
+#: src/text.vala:339
+msgid "C_onnect"
+msgstr "_Conectar"
+
+#: src/text.vala:340
+msgid "_Disconnect"
+msgstr "_Desconectar"
+
+#: src/text.vala:341
+msgid "Reconnect"
+msgstr "Reconectar"
+
+#: src/text.vala:342
+msgid "Connected"
+msgstr "Conectado"
+
+#: src/text.vala:343
+msgid "Disconnected"
+msgstr "Desconectado"
+
+#: src/text.vala:344
+msgid "Initializing…"
+msgstr "A inicializar..."
```

Comment:
Unicode ellipsis character `…` should be used instead of three periods. Easiest thing to do is find/replace `...` with `…` everywhere in this file.

Reply:
Done